contracts/axelarnet-gateway/src/lib.rs

```diff
@@ -6,4 +6,4 @@ mod state;
 mod clients;
 pub use clients::external::AxelarExecutableMsg;
 pub use clients::gateway::Client;
-pub use state::ExecutableMessage;
+pub use state::{Error as StateError, ExecutableMessage};
```

Comment:
decided to expose StateError since some failures are harder to distinguish otherwise